packages/v4/src/pages/index.js

```diff
@@ -119,10 +121,9 @@ const IndexPage = ({ location }) => {
                   </Text>
                 </div>
                 <div className="pf-u-pb-2xl">
-                  <Text component={TextVariants.h2}>Start building with components</Text>
+                  <Text component={TextVariants.h2}>Build things your way</Text>
                   <Text component={TextVariants.p} className="ws-mdx-p">
-                    Components are like building blocks. Designed to be flexible and modular, you can mix and match to
-                    create a solution for almost any UI problem.
+                    Components are like building blocks. Designed to be flexible and modular, you can mix and match to create a solution for almost any UI problem.
```

Comment:
My expectation in reading this sentence would be:
"Components are like building blocks. Designed to be flexible and modular, you can mix and match components to create a solution for almost any UI problem." Just added another "components" in here for clarity.

Reply:
Good idea!